### PR TITLE
update to 0.4.20

### PR DESCRIPTION
**Pull request by cbouss**
update to 0.4.20

https://github.com/Thriftpy/thriftpy2/tree/v0.4.20

### CODE REVIEW

**Review by SC426 — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -21,15 +21,15 @@ requirements:
     - python
     - setuptools
     - wheel
+    - cython >=0.28.4,<4
   run:
     - ply >=3.4,<4.0
     - python
-    - six 1.15.*
+    - six >=1.15,<2.dev0
```
Comment (boldorider4) [inline recipe/meta.yaml:28]: should this not be `2.0a0`?

Comment (Jrice1317) [reply recipe/meta.yaml:28]: I'm not sure the difference between those two, but maybe `2.*` would work best and catch all cases?

Comment (Jrice1317) [reply recipe/meta.yaml:28]: I'm not sure the difference between those two, but maybe `2.*` would work best and catch all cases?

Comment (cbouss) [reply recipe/meta.yaml:28]: that was grayskull output, but it's correct.
https://github.com/conda/conda/blob/main/conda/models/version.py#L110
_'dev' versions are smaller than all corresponding versions of other types_

**Pull request merged**